components/toolbar-menu/index.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
We will encounter some keyboard interactivity requirements with this component similar to those targeted at the Inserter (#578). The two components are similar in many ways. In a future pull request we might even consider consolidating them, or generalizing the keyboard interactivity, if not at least recreating the effect here.